### ISSUE TITLE
KeyError: 'ip'

### ISSUE THREAD

**Issue by kurdtpage** [nicolargo/glances#1842]
**Describe the bug**
Cannot run glances, fresh install

**To Reproduce**
1. `sudo apt-get install glances`
2. `glances`
3. See error

**Expected behavior**
Should run properly with no paramters given, or should display required parameters if any

**Screenshots**
```
Traceback (most recent call last):
File "/usr/bin/glances", line 11, in <module>
load_entry_point('Glances==3.1.3', 'console_scripts', 'glances')()
File "/usr/lib/python3/dist-packages/glances/__init__.py", line 143, in main
start(config=config, args=args)
File "/usr/lib/python3/dist-packages/glances/__init__.py", line 112, in start
mode.serve_forever()
File "/usr/lib/python3/dist-packages/glances/standalone.py", line 151, in serve_forever
loop = self.__serve_forever()
File "/usr/lib/python3/dist-packages/glances/standalone.py", line 138, in __serve_forever
ret = not self.screen.update(self.stats, duration=adapted_refresh)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 982, in update
self.flush(stats, cs_status=cs_status)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 957, in flush
self.display(stats, cs_status=cs_status)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 581, in display
self.__display_header(__stat_display)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 646, in __display_header
self.display_plugin(stat_display["ip"])
KeyError: 'ip'
Error in sys.excepthook:
Traceback (most recent call last):
File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_bin_glances.1000.crash'

Original exception was:
Traceback (most recent call last):
File "/usr/bin/glances", line 11, in <module>
load_entry_point('Glances==3.1.3', 'console_scripts', 'glances')()
File "/usr/lib/python3/dist-packages/glances/__init__.py", line 143, in main
start(config=config, args=args)
File "/usr/lib/python3/dist-packages/glances/__init__.py", line 112, in start
mode.serve_forever()
File "/usr/lib/python3/dist-packages/glances/standalone.py", line 151, in serve_forever
loop = self.__serve_forever()
File "/usr/lib/python3/dist-packages/glances/standalone.py", line 138, in __serve_forever
ret = not self.screen.update(self.stats, duration=adapted_refresh)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 982, in update
self.flush(stats, cs_status=cs_status)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 957, in flush
self.display(stats, cs_status=cs_status)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 581, in display
self.__display_header(__stat_display)
File "/usr/lib/python3/dist-packages/glances/outputs/glances_curses.py", line 646, in __display_header
self.display_plugin(stat_display["ip"])
KeyError: 'ip'
```

**Desktop (please complete the following information):**
 - OS: Linux Mint 20.1
 - Glances Version: Glances v3.1.3 with psutil v5.5.1
 - Glances logs file: No log file could be found

**Additional context**
Running as standard user, no sudo

**Comment by kurdtpage** [comment]
Oddly, it produces the above error if the CLI is windowed (640*480 or so) but works fine if the window is maximized (1920*1080 or so)

**Comment by nicolargo** [comment]
Already corrected in Glances 3.1.4 or higher (see patch: https://github.com/nicolargo/glances/commit/1d6be6ceabf79425e9b92caff57a655cb252d66b)

Please upgrade.

**Comment by kfiresmith** [comment]
I think this regressed.  I see it all the time in the latest Snap (3.1.5) on Ubuntu 20.
